pkgs/tools/networking/speedtest-go/default.nix

```diff
@@ -1,22 +1,22 @@
-{ lib
-, buildGoModule
-, fetchFromGitHub
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
 }:
-
 buildGoModule rec {
   pname = "speedtest-go";
-  version = "1.7.0";
+  version = "1.7.5";
 
   src = fetchFromGitHub {
     owner = "showwin";
     repo = pname;
     rev = "refs/tags/v${version}";
-    hash = "sha256-G/ovJk/4pDSDlOUSP9/d0vzTg5IaMPMMPhAN+VGxi/c=";
+    hash = "sha256-8YN7k7sP0S3MIArUfwDYMQvcCKNTxauQjevyOoisAuc=";
   };
 
   vendorHash = "sha256-wQqAX7YuxxTiMWmV9LRoXunGMMzs12UyHbf4VvbQF1E=";
```

Comment:
Are you sure there's no vendorHash update?

Reply:
It didn't update it. I double checked changing the vendorHash using the `lib.fakeHash`.

Reply:
Checking the repository the owner didn't update for about a year.

To double check a possibile change i updated the vendorHash value using the `lib.fakeHash` to see what was the real value.